server/graphql/v2/object/Individual.js

```diff
@@ -108,11 +108,28 @@ export const Individual = new GraphQLObjectType({
           }
         },
       },
-      changelogViewDate: {
-        type: GraphQLDateTime,
-        async resolve(collective) {
-          const user = await collective.getUser();
-          return user?.changelogViewDate;
+      hasSeenLatestChangelogEntry: {
+        type: GraphQLBoolean,
+        async resolve(individual) {
+          const cacheKey = 'latest_changelog_publish_date';
+          let latestChangelogUpdate = await cache.get(cacheKey);
+          if (!latestChangelogUpdate) {
+            const collectiveId = await fetchCollectiveId('opencollective');
+            latestChangelogUpdate = await models.Update.findOne({
+              where: {
+                CollectiveId: collectiveId,
+                publishedAt: { [Op.ne]: null },
+                isChangelog: true,
+              },
+              order: [['publishedAt', 'DESC']],
+              limit: 1,
+            });
+
+            // keep the latest change log publish date for a day in cache
+            cache.set(cacheKey, latestChangelogUpdate, 24 * 60 * 60);
```

Comment:
No need to store the full update in the cache

```es6
cache.set(cacheKey, latestChangelogUpdate?.publishedAt, 24 * 60 * 60);
```

Reply:
Thanks. Done. 👍🏽